arch/arm/src/stm32h7/Kconfig

```diff
@@ -41,6 +41,7 @@ config STM32H7_STM32H7X3XX
 	select ARMV7M_HAVE_ITCM
 	select ARMV7M_HAVE_DTCM
 	select STM32H7_HAVE_ETHERNET
+	select STM32H7_HAVE_FMC
```

Comment:
Hi, from what I see the whole STM32H7 family has support for FMC, so probably we can enable this feature also for STM32H7_STM32H7X7XX